doc: improve user guidance (#7357)
nrfconnect/sdk-nrf: doc/nrf/ug_nrf9160_gs.rst

This section will get you started with your nRF9160 DK.
You will update the firmware (both the application firmware and the modem firmware) and the nRF Cloud certificates of the DK, and conduct some initial tests.

> If you have already set up your nRF9160 DK and want to learn more:
> 
> * If you are already familiar with the |NCS|, see :ref:`ug_nrf9160` for more advanced topics related to the nRF9160 DK.
> * If you are not familiar with the |NCS| and the development environment, see the :ref:`introductory documentation <getting_started>`.

Reviewer: ```suggestion
If you have already set up your nRF9160 DK and want to learn more, see the following documentation:

* :ref:`introductory documentation <getting_started>` for more information on |NCS| and the development environment.
* :ref:`ug_nrf9160` for more advanced topics related to the nRF9160 DK if you are already familiar with the |NCS|.

```

#. From the :guilabel:`Devices` view, open the entry for your device.
#. Observe that after a while, the GNSS data is displayed on the map in the :guilabel:`GPS Data` card on nRF Cloud.

Next steps
**********

You have now completed getting started with the nRF9160 DK.
See the following links for where to go next:

> * If you are already familiar with the |NCS|, see :ref:`ug_nrf9160` for more advanced topics related to the nRF9160 DK.
> * If you are not familiar with the |NCS| and the development environment, see the :ref:`introductory documentation <getting_started>`.

Reviewer: ```suggestion
* :ref:`introductory documentation <getting_started>` for more information on |NCS| and the development environment.
* :ref:`ug_nrf9160` for more advanced topics related to the nRF9160 DK if you are already familiar with the |NCS|.
```
